src/modules/generators/fractals/basicmulti.rs

```diff
@@ -0,0 +1,226 @@
+// Copyright 2016 The Noise-rs Developers.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use num_traits::Float;
+use math;
+use math::{Point2, Point3, Point4};
+use NoiseModule;
+use modules::Perlin;
+use super::util::*;
```

Comment:
Would kinda prefer we not glob import - made it hard to find where `build_sources` was from - was looking in this module. I'd prefer `util::build_sources`